andela/travel_tool_front
# #161983423 Fix checkin/checkout UI design

#### What does this PR do?
Fixes the check-ins and checkout UI design

#### Description of Task to be completed?
- It expected that the UI design that holds the checked-in / checked-out status aligns with what is on the invision mockup. Attention is paid to the borders on the left and right.

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- checkout the branch `git checkout bg-change-guest-room-161744068`
- Start the server with `make start`
- Start the backend of this project `git clone https://github.com/andela/travel_tool_back.git && make start`
- Visit https://travela-local.andela.com
- Login to the application
- Ensure you have a travel admin privilege
- Add a request and approve in
- Navigate to the residence `Check-in` page
- The check-ins should show as in the mockup

#### Any background context you want to provide?
- You should have a few requests on the system so you can view the check-ins

#### What are the relevant pivotal tracker stories?
[#161983423](https://www.pivotaltracker.com/story/show/161983423)

#### Screenshots (if appropriate)

### Before
![image](https://user-images.githubusercontent.com/30747298/48565495-bd9d9a00-e909-11e8-8399-39378af4a10e.png)

### After
![image](https://user-images.githubusercontent.com/30747298/48565524-cf7f3d00-e909-11e8-9546-6b0b6b679ebb.png)



#### Questions:
